packages/csv-parser-order/test/add-return-info.spec.js

```diff
@@ -0,0 +1,187 @@
+import fs from 'fs'
+import StreamTest from 'streamtest'
+import path from 'path'
+
+import AddReturnInfoParser from '../src/parsers/add-return-info'
+
+describe('AddReturnInfoParser', () => {
+  it('should be a class', () => {
+    expect(typeof AddReturnInfoParser).toBe('function')
```

Comment:
From the message, I am not sure this assertion is valid, does not ensure it's a class (_a JS issue though_) 😒